env/Tensorflow_probability.yml

```diff
@@ -0,0 +1,33 @@
+name: Tensorflow
+channels:
+  - conda-main
+  - conda-mo-internal
+  - conda-forge
+  - conda-r
+dependencies:
+  - python=3.7
```

Comment:
python 3.7 is getting on a bit now, have you tried at least a version 3.8 environment?

Reply:
I haven't tried 3.8, but I can't imagine it will be a problem. I'll maybe test that next week and do a separate PR

Reply:
good idea, get this one merged and think about py3.8 separately.